lib/livebook/runtime/attached.ex

```diff
@@ -90,7 +86,10 @@ defimpl Livebook.Runtime, for: Livebook.Runtime.Attached do
     )
   end
 
-  def duplicate(_runtime) do
-    {:error, "attached runtime is connected to a specific VM and cannot be duplicated"}
+  def duplicate(runtime) do
+    case Livebook.Runtime.Attached.init(runtime.node, runtime.cookie) do
+      {:ok, runtime} -> {:ok, runtime}
+      {:error, :unreachable} -> "node #{inspect(runtime.node)} is unreachable"
```

Comment:
Is returning a string expected here? Or should it be `{:error, msg}`?

Reply:
Whoops, good catch!